Adding Papiamento translation of README file (#4462)
the-turing-way/the-turing-way: README-translated/README-Papiamento.md

| **Site** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

*Bon bini na e repositorio di Github di e proyecto The Turing Way. Aki ta unda tur componente di e proyecto ta desaroya, revisa y mantene.*

_The Turing Way_ ta un guia practico pa data science reproducibel, etico y colaborativo.
Nos ta support un comunidad diverso di colaboradornan pa logra un data science accesibel, comprensibel y efectivo pa tur hende.

> Nos meta ta di provee tur informacion cu investigadornan y data scientist den academia, industria y sector publico mester na cuminsamento di nan proyectonan pa asegura cu nan ta fasil pa reproduci na final.

Reviewer: ```suggestion
Nos meta ta di provee tur informacion cu investigador y data scientist den academia, industria y sector publico mester na cuminsamento di nan proyectonan pa asegura cu nan ta facil cu esey pa reproduci na final.
```

[![E proyecto The Turing Way ta un buki, un comunidad, un proyecto open source y un cultura di colaboracion. Esaki ta mustra den cuater ilustracion, e prome ta mustra e buki di Turing Way, e di dos ta mustra con e comunidad por crese, e di tres ta mustra dos persona colaborando riba un pull request, y e ultimo ta mustra e balansa unda reproducibilidad ta balora mas cu e cantidad di papers publica](../book/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).

*The Turing Way ta un buki, un comunidad y un colaboracion global.*

> Tur partido interesa, incluiendo studiante, investigador, ingeniero di software, lider di proyecto y ekipo fundacional, ta invita pa uza The Turing Way pa compronde nan rol y nan responsabilidad den reproducibilidad di data science.

Reviewer: ```suggestion
Tur partido interesa, incluyendo studiante, investigador, ingeniero di software, y lider di proyecto, ta invita pa uza The Turing Way pa compronde nan rol y nan responsabilidad den reproducibilidad di data science.
```

[![Screenshot di e buki online di Turing Way. Tambe ta mustra un di e ilustracionnan di Turing Way na cuminsamento di e buki. Den e ilustracion aki, nos por mira un caminda of highway cu tiendanan pa diferente habilidadnan di data science. Hende por drenta y sali cu nan lista di compra y scohe loke nan mester.](../book/figures/README_imgs/README_book.png)](https://book.the-turing-way.org/welcome.html)

*Screenshot di e buki online di The Turing Way ([uza e imagen aki den un presentacion](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*

The Turing Way a cuminsa na 2019 como un guia chikito di data science, y for di esaki a expande pa forma un manual hopi mas completo forma di varios guia cu ta cubri varios tema: Investigacion Reproducibel, Diseño di Proyecto, Comunicacion, Colaboracion y Investigacion Etico.
Cada guia ta ofrece capitulo riba un variedad di tema cu ta trata bon practica y recomendacion.

> E capitulonan aki a wordo co-scribi door di personanan cu ta colabora y cu ta studiante, investigador, docente, lider comunitario, politico y profesionalnan di diferente background, experencia biba y conocemento di dominio.

Reviewer: ```suggestion
E capitulonan aki a wordo co-skirbi pa personanan pa colaborador, studiante, investigador, docente, lider comunitario, politico y profesional di diferente background, experencia biba y conocemento di dominio.
```

| **Site** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

*Bonbini na e repositorio di GitHub di e proyecto The Turing Way. 
Aki ta tur componente di e proyecto ta wordo desaroya, revisa y mantene.*

_The Turing Way_ ta un guia practico pa data science reproducibel, etico y colaborativo.
Nos ta support un comunidad diverso di colaboradornan pa logra un data science accesibel, comprensibel y efectivo pa tur hende.

> Nos meta ta di provee tur informacion cu investigador y data scientist den academia, industria y sector publico mester na cuminsamento di nan proyectonan pa asegura cu nan ta facil cu esey pa reproduci na final.

Reviewer: ```suggestion
Nos meta ta pa provee tur informacion cu investigador y data scientist den academia, industria y sector publico mester na cuminsamento di nan proyectonan pa asegura cu nan ta facil cu esey pa reproduci na final.
```

Cada guia ta ofrece capitulo riba un variedad di tema cu ta trata bon practica y recomendacion.
E capitulonan aki a wordo co-skirbi pa personanan pa colaborador, studiante, investigador, docente, lider comunitario, politico y profesional di diferente background, experencia biba y conocemento di dominio.

***Nos meta ta pa haci reproducibilidad "demasiado facil pa no hacie".***

**Tabla di contenido:**

- [Tocante e proyecto](#tocante-e-proyecto)
- [E ekipo](#e-ekipo)
- [Contribui](#contribui)
- [Site _The Turing Way_](#site-the-turing-way)
- [Tuma contacto cu nos](#tuma-contacto-cu-nos)
- [Colaboradornan](#Colaboradornan)

🎧 Si bo ta prefera un introduccion di audio di e proyecto (na Ingles) Rachael, un colaborador di e proyecto a presenta na [Open Science Fair 2019](https://www.opensciencefair.eu/) na Porto y su presentacion a wordo graba door di [Podcast Orion](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
E descripcion general di _The Turing Way_ ta cuminsa na minuut 5:13.

### Tocante e proyecto

Investigacion reproducibel ta nesesario pa garantisa cu bo por confia den trabou cientifico.

> Esnan cu ta finansia y edit trabounan cientifico ta cuminsa exihi cu publicacionnan inclui acceso na e data uza y e codigo cu cua nan a haci e analisis.

Reviewer: ```suggestion
Esnan cu ta financia y edit trabounan cientifico ta cuminsa exigi cu publicacionnan tin cu inclui acceso na e data uza y e codigo cu cua nan a haci e analisis.
```

E meta ta pa garantisa cu tur resultadonan por wordo verifica di forma independiente y por sirbi como base pa trabou futuro.
Tin biaha esaki ta mas facil pa bisa cu pa haci.
Comparti e resultadonan di investigacion aki ta nifica compronde maneho di data, biblioteconomi, desaroyo di software y technicanan di integracion continuo: habilidadnan cu no ta duna lesnan di dje ni ta spera di investigador academico y data scientist.
Mirando cu e actividadnan aki no ta duna lesnan di dje, nos ta reconoce cu e carga di e exsigencianan y haya  habilidadnan nobo por ta intimidante pa personanan cu ta nobo den e mundo aki.

> *The Turing Way* ta un manual pa yuda esnan cu ta studia, supervisa, financia y edit trabounan cientifico pa garantisa cu data science reproducibel ta "demasiado facil pa no haci'e" inclui pa personanan cu nunca prome a traha di e manera aki.

Reviewer: ```suggestion
*The Turing Way* ta un manual pa yuda esnan cu ta studia, supervisa, financia y edit trabounan cientifico pa garantisa cu data science reproducibel ta "demasiado facil pa no hacie" inclui pa personanan cu nunca prome a traha di e manera aki.
```

### E ekipo

_The Turing Way_ ta un proyecto di colaboracion habri y impulsa door di comunidad.
Tur esnan cu ta colabora den e buki aki, indiferente di con chikito of grandi cu nan contribucion ta, ta reconoce den e proyecto aki como colaborador y miembro di comunidad.
E colaboradornan di largo duracion ta considera parti di e grupo di colaborador principal, cua ta asumi rol di lider den e proyecto.

E proyecto ta coordina door di lider investigador **Kirstie Whitaker** (fundador) y **Malvika Sharan** (desaroyador di comunidad) y ta hospeda na [The Alan Turing Institute](https://www.turing.ac.uk/).

Bo por lesa con ta haci [e proceso di reconosemento como parti di _The Turing Way_](https://book.the-turing-way.org/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) y e [Registro di Colaboradornan](https://book.the-turing-way.org/afterword/contributors-record.html) pa conoce con ta haci e reconocimento di trabou y nos Colaboradornan destaca.
Por fabor bishita e [Panel di Colaboradornan](#Colaboradornan) pa mira e profil di GitHub di tur nos colaboradornan.

### Contribui

> :construction: E repositorio aki ta un trabou den progreso continuo y **tur hende** ta invita pa yuda nos construi algo cu ta util pa otro hendenan. :construction:

Reviewer: ```suggestion
:construction: E repositorio aki ta un trabou den progreso continuo y **tur hende** ta invita pa yuda nos construi algo cu ta util pa otro hende. :construction:
```

> Ta spera cu tur personanan cu kier uni na e proyecto sigui nos [código di conducta](../CODE_OF_CONDUCT.md) y cu bishita nos [guia pa contribui na e proyecto](../CONTRIBUTING.md) pa mas informacion tocante con pa cuminsa.

Reviewer: ```suggestion
Ta spera cu tur personanan cu kier uni na e proyecto sigui nos [codigo di conducta](../CODE_OF_CONDUCT.md) y cu bishita nos [guia pa contribui na e proyecto](../CONTRIBUTING.md) pa mas informacion tocante con pa cuminsa.
```

> Nos kier conoce nos colaboradornan ken nan ta.

Reviewer: ```suggestion
Nos kier conoce nos colaboradornan, nos kier sa ken nan ta.
```

Nos a crea un [pakete di promocion](https://github.com/the-turing-way/the-turing-way/tree/main/communications/promotion-pack) pa yuda bo presenta y comparti The Turing Way den bo red.

### Sita _The Turing Way_

Bo por haci referencia na _The Turing Way_ pa medio di e archivo Zenodo di e proyecto uzando DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
DOI ta permiti nos archiva e repositorio y ta realmente valioso pa garantisa cu e trabou ta rastrea den publicacionnan academico.

E sitacion lo tin un aspecto paresido na esaki (na Ingles):

> The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. <http://doi.org/10.5281/zenodo.3233986>

Tambe bo por comparti e URL cu hende por lesa na un pagina di e buki, por ehempel, <https://book.the-turing-way.org/reproducibility/03/definitions.html>, pero conta cu e proyecto ta den desaroyo y, pues, e enlacenan aki por cambia cu tempo.

> Por ta cu bo kier inclui un enlace di archivo web manera: <https://web.archive.org/web/20191030093753/https://book.the-turing-way.org/reproducibility/03/definitions.html> pa asegura cu bo no termina cu enlace kibra na tur banda.

Reviewer: ```suggestion
Por ta cu bo kier inclui un enlace di archivo web manera: <https://web.archive.org/web/20191030093753/https://book.the-turing-way.org/reproducibility/03/definitions.html> pa asegura cu bo no ta haya bo cu enlace kibra.
```

E ilustracionnan aki ta disponibel na Zenodo ([https://zenodo.org/record/13882307)](https://zenodo.org/record/13882307) bou un licencia CC-BY.

Ora bo uza cualkier di e imagennan, por fabor inclui e atribucion aki:
> E imagen aki a wordo crea dor di Scriberia pa comunidad The Turing Way y ta wordo uza bou un licencia CC-BY.

E ultimo version di Zenodo por wordo cita como:
> The Turing Way Community, & Scriberia. (2024). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.13882307

Nos a uza algun di e ilustracionnan aki den e responsnan na e contribucionnan di personanan cu ta integra na e comunidad aki den e [Bot di bon bini](https://github.com/apps/welcome) di e repositorio di GitHub aki.

### Tuma contacto cu nos

#### Email

Bo por tuma contacto cu e ekipo di *The Turing Way* mandando un email na [theturingway@gmail.com](mailto:theturingway@gmail.com) of [turingway@turing.ac.uk](mailto:turingway@turing.ac.uk).

#### Chat

Conecta cu otro hendenan y comparti bo ideanan na Slack uzando [e enlace di invitacion aki](https://join.slack.com/t/theturingway/shared_invite/zt-2v7euwuo7-BYstHdKuTNd1ce0puDtBxA).

#### Ricibi actualisacion

> Tambe nos tin un lista di mail na cua nos ta manda actualisacionnan mensual di e proyecto.

Reviewer: ```suggestion
Tambe nos tin un lista di mail na cua nos ta manda actualisacion mensual di e proyecto.
```

> Subscribe na <https://buttondown.com/turingway>.

Reviewer: ```suggestion
Subscribe riba <https://buttondown.com/turingway>.
```

| **Construi** | [![Netlify build](https://github.com/the-turing-way/the-turing-way/workflows/CI/badge.svg)](https://github.com/the-turing-way/the-turing-way/actions?query=workflow%3ACI+branch%3Amain) [![Latin Phrases](https://github.com/the-turing-way/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/the-turing-way/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amain) [![TestFileSizes](https://github.com/the-turing-way/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/the-turing-way/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amain) [![Check for Lorem Ipsums](https://github.com/the-turing-way/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/the-turing-way/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amain) |
| **Site** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

> *Bon bini na e repositorio di Github di e proyecto The Turing Way. Aki ta unda tur componente di e proyecto ta desaroya, revisa y mantene.*

Reviewer: ```suggestion
*Bon bini na e repositorio di GitHub di e proyecto The Turing Way. 
Aki ta tur componente di e proyecto ta desaroya, revisa y mantene.*
```

Nos ta support un comunidad diverso di colaboradornan pa logra un data science accesibel, comprensibel y efectivo pa tur hende.
Nos meta ta di provee tur Informacion cu investigadornan y data scientist den academia, industria y sector publico mester na cuminsamento di nan proyectonan pa asegura cu nan ta fasil pa reproduci na final.

[![E proyecto The Turing Way ta un buki, un comunidad, un proyecto open source y un cultura di colaboracion. Esaki ta mustra den cuater ilustracion, e prome ta mustra e buki di Turing Way, e di dos ta mustra con e comunidad por crese, e di tres ta mustra dos persona colaborando riba un pull request, y e ultimo ta mustra e balansa unda reproducibilidad ta balora mas cu e cantidad di papers publica](../book/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).

*The Turing Way ta un buki, un comunidad y un colaboracion global.*

Tur partido interesa, incluiendo studiante, investigador, ingeniero di software, lider di proyecto y ekipo fundacional, ta invita pa usa The Turing Way pa compronde nan rol y nan responsabilidad den reproducibilidad di data science.

> Bo por lesa e buki [online](https://github.com/the-turing-way/the-turing-way), contribui na e proyecto manera ta splica den nos [guia pa contribui na e proyecto](https://github.com/the-turing-way/the-turing-way/blob/main/CONTRIBUTING.md) y re-utilisa tur material ([mira e Licencia](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)).

Reviewer: ```suggestion
Bo por lesa e buki [online](https://github.com/the-turing-way/the-turing-way), contribui na e proyecto manera ta splica den nos [guia pa contribui na e proyecto](https://github.com/the-turing-way/the-turing-way/blob/main/CONTRIBUTING.md) y re-utiliza tur material ([mira e Licencia](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)).
```

[![Screenshot di e buki online di Turing Way. Tambe ta mustra un di e ilustracionnan di Turing Way na cuminsamento di e buki. Den e ilustracion aki, nos por mira un caminda of highway cu tiendanan pa diferente habilidadnan di data science. Hende por drenta y sali cu nan lista di compra y scohe loke nan mester.](../book/figures/README_imgs/README_book.png)](https://book.the-turing-way.org/welcome.html)

*Screenshot di e buki online di The Turing Way ([usa e imagen aki den un presentacion](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*

The Turing Way a cuminsa na 2019 como un guia chikito di data science, y for di esaki a expande pa forma un manual hopi mas completo forma di varios guia ku ta cubri varios tema: Investigacion Reproducibel, Diseño di Proyecto, Comunicacion, Colaboracion y Investigacion Etico.
Cada guia ta ofrece capitulo riba un variedad di tema cu ta trata bon practica y recomendacion.

> E capitulonan aki a wordo co-scribi door di personanan cu ta colabora y cu ta studiante, investigador, docente, lider comunitario, politico y profesionalnan di diferente background, experencia biba y conosemento di dominio.

Reviewer: ```suggestion
E capitulonan aki a wordo co-scribi door di personanan cu ta colabora y cu ta studiante, investigador, docente, lider comunitario, politico y profesionalnan di diferente background, experencia biba y conocemento di dominio.
```

***Nos meta ta di hasi reproducibilidad "demasiado fasil pa no hasie".***

**Tabla di contenido:**

- [Tocante e proyecto](#tocante-e-proyecto)
- [E ekipo](#e-ekipo)
- [Contribui](#contribui)
- [Site _The Turing Way_](#site-the-turing-way)
- [Tuma contacto cu nos](#tuma-contacto-cu-nos)
- [Colaboradornan](#Colaboradornan)

🎧 Si bo ta prefera un introducion audio na e proyecto (na ingles) Rachel, un colaborador di e proyecto a presenta na [Open Science Fair 2019](https://www.opensciencefair.eu/) na Porto y su presentacion a wordo graba door di [Podcast Orion](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
E descripcion general di _The Turing Way_ ta cuminsa na minuut 5:13.

### Tocante e proyecto

Investigacion reproducibel ta nesesario pa garantisa cu bo por confia den trabou cientifico.
Esnan cu ta finansia y edit trabounan cientifico ta cuminsa exihi cu publicacionnan inclui acceso na e data usa y e codigo cu cua nan a hasi e analisis.
E meta ta pa garantisa cu tur resultadonan por wordo verifica di forma independiente y por sirbi como base pa trabou futuro.

> Tin biaha esaki ta mas fasil pa bisa cu pa hasi.

Reviewer: ```suggestion
Tin biaha esaki ta mas facil pa bisa cu pa haci.
```

Comparti e resultadonan di investigacion aki ta nifica compronde maneho di data, biblioteconomi, desaroyo di software y technicanan di integracion continuo: habilidadnan cu no ta duna lesnan di dje ni ta spera di investigador academico y data scientist.

> Mirando cu e actividadnan aki no ta duna lesnan di dje, nos ta reconoce cu e carga di e exsigencianan y haya  habilidadnan nobo por ta intimidante pa personanan cu ta nobo den e mundo aki.

Reviewer: ```suggestion
Mirando cu e actividadnan aki no ta duna lesnan di dje, nos ta reconoce cu e carga di e rekisitonan y haya habilidadnan nobo por ta intimidante pa personanan cu ta nobo den e mundo aki.
```

### E ekipo

_The Turing Way_ ta un proyecto di colaboracion habri y impulsa door di comunidad.
Tur esnan cu ta colabora den e buki aki, indiferente di con chikito of grandi cu nan contribucion ta, ta reconoce den e proyecto aki como colaborador y miembro di comunidad.
E colaboradornan di largo duracion ta considera parti di e grupo di colaborador principal, cua ta asumi rol di lider den e proyecto.

E proyecto ta coordina door di lider investigador **Kirstie Whitaker** (fundador) y **Malvika Sharan** (desaroyador di comunidad) y ta hospeda na [The Alan Turing Institute](https://www.turing.ac.uk/).

Bo por lesa con ta hasi [e proceso di reconosemento como parti di _The Turing Way_](https://book.the-turing-way.org/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) y e [Registro di Colaboradornan](https://book.the-turing-way.org/afterword/contributors-record.html) pa conoce con ta hasi e reconocimento di trabou y nos Colaboradornan destaca.
Por fabor bishita e [Panel di Colaboradornan](#Colaboradornan) pa mira e profil di Github di tur nos colaboradornan.

### Contribui

> :construction: E repositorio aki ta un trabou den progreso continuo y **tur hende** ta invita pa yuda nos construí algo cu ta util pa otro hendenan. :construction:

Reviewer: ```suggestion
:construction: E repositorio aki ta un trabou den progreso continuo y **tur hende** ta invita pa yuda nos construi algo cu ta util pa otro hendenan. :construction:
```

Nos kier conose nos colaboradornan ken nan ta.
Pa eset, nos ta provee varios punto di entrada pa bo por colabora basa riba bo interes, disponibilidad of habilidadnan.

![E imagen aki ta mustra seis tipo di contribucion cu cualke hende por hasi. Esakinan ta: Desaroyay komparti, Mantene y mehora, Compartí recurso, Revisa y actualisa, Hasie global pa medio di traducion, y Comparti e miho practicanan](../book/figures/README_imgs/README_contributions.png)

E contribucionnan ta inclui e desaroyo y difucion di nobo capitulonan; mantencion y mehora capitulonan exsistente; compartí recurso di _The Turing Way_; revisa y actualisa material desaroya anteriormente; traduci e capitulonan pa yuda hasi e proyecto aki acsesibel di forma global, y compartí e miho prakticanan den investigashon.

E miembronan di comunidad tin oportunidad pa siña habilidadnan nobo, comparti nan ideanan y colabora entre nan mes.
Nan tambe tin oportunidad di mentoria den e proyecto mientras nan ta hasi nan contribucionnan den The Turing Way of otro proyectonan di codigo habri y ta invita pa guia nobo participante den e proyecto.

> Nos a crea un [pakete di promoshon](https://github.com/the-turing-way/the-turing-way/tree/main/communications/promotion-pack) pa yuda bo presenta y compartiThe Turing Way den bo red.

Reviewer: ```suggestion
Nos a crea un [pakete di promocion](https://github.com/the-turing-way/the-turing-way/tree/main/communications/promotion-pack) pa yuda bo presenta y comparti The Turing Way den bo red.
```

> ### Sita _The Turing Way_

Reviewer: ```suggestion
### Cita _The Turing Way_
```

Nan ta representa un variedad di contenido di e manual, di e esfuersonan di colaboracion den comunidad y di e proyecto _The Turing Way_ en general.
E ilustracionnan aki ta disponibel na Zenodo ([https://zenodo.org/record/13882307)](https://zenodo.org/record/13882307) bou un lisensia CC-BY.

Ora bo usa cualkier di e imagennan, por fabor inclui e atribucion aki:
> E imagen aki a wordo crea dor di Scriberia pa comunidad The Turing Way y ta wordo usa bou un lisensia CC-BY.

E ultimo verscion di Zenodo por wordu cita como:
> The Turing Way Community, & Scriberia. (2024). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.13882307

Nos a usa algun di e ilustracionnan aki den e responsnan na e contribucionnan di personanan cu ta integra na e comunidad aki den e [Bot di bon bini](https://github.com/apps/welcome) di e repositorio di GitHub aki.

### Tuma contacto cu nos

#### Email

Bo por tuma contacto cu e ekipo di *The Turing Way* mandando un email na [theturingway@gmail.com](mailto:theturingway@gmail.com) of [turingway@turing.ac.uk](mailto:turingway@turing.ac.uk).

#### Chat

> Conecta cu otro hendenan y comparti bo ideanan na Slack usando [e enlace di invitacion aki](https://join.slack.com/t/theturingway/shared_invite/zt-2v7euwuo7-BYstHdKuTNd1ce0puDtBxA).

Author: Usando is with an s

> Nos tin un sala di chat di Gitter (si bo ta prefera un alternativa di codigo habri pa chat) y nos lo ta contento si bo pasa pa saluda na <https://gitter.im/the-turing-way/the-turing-way>.

Reviewer: ```suggestion
```

Reviewer: We actually no longer use that.... oops
